examples/warning-should/ShadowedTypeVarClass.purs

```diff
@@ -0,0 +1,5 @@
+-- @shouldWarnWith ShadowedTypeVar
```

Comment:
Yeah, this is something I discovered that I figure should warn, but we currently don't catch. I made the warning detection like the errors: it expect an exact set of warnings, so yeah, it would break the tests currently.

Perhaps I should just open an issue instead and delete this.